src/App.js

```diff
@@ -35,6 +37,8 @@ function SihWizard (){
         <ContentWrapper>
             <Routes>
                 <Route path="/" element={ <Welcome/> } />
+                <Route path="/login" element={ <LoginForm/> } />
+                <Route path="/wizard" element={ <Authenticate> <CarePlanWizard /> </Authenticate> } />
                 <Route path="/settings" element={ <Authenticate> <Settings /> </Authenticate> } />
```

Comment:
@hawaism no need to add login and wizard route

Reply:
resolved